src/track/track.js

```diff
@@ -0,0 +1,33 @@
+//@flow
+export default class Track {
+  _id: string;
+  _active: boolean;
+  _type: string;
+  _language: string;
+
+  get id(): string {
+    return this._id;
+  }
+
+  get active(): boolean {
+    return this._active;
+  }
+
+  set active(value: boolean) {
+    this._active = value;
+  }
+
+  get type(): string {
+    return this._type;
+  }
+
+  get language(): string {
+    return this._language;
+  }
+
+  constructor(id: string, active: boolean, language: string) {
```

Comment:
I think it's generally better to send a _details_ object that will contain all the data.
In this way we don't need to care about the order and the amount of the params .
For example:
`constructor(details: Object) {`
   ` this._id = details.id;`
   ` this._active = details.active;`
   ` this._name = details.name;`
`  }`